colour-codes.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
```

Comment:
Should add `# shellcheck disable=SC2034` to tell the lint checker to ignore unused variables errors, as this file is intended to be included in other scripts.